Userland/Applications/CrashReporter/main.cpp

```diff
@@ -317,12 +317,15 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     save_backtrace_button.set_enabled(false);
 
     (void)Threading::BackgroundAction<ThreadBacktracesAndCpuRegisters>::construct(
-        [&, coredump = move(coredump)](auto&) {
+        [&, window = window->make_weak_ptr<GUI::Window>(), coredump = move(coredump)](auto&) {
             ThreadBacktracesAndCpuRegisters results;
             size_t thread_index = 0;
             coredump->for_each_thread_info([&](auto& thread_info) {
                 results.thread_backtraces.append(build_backtrace(*coredump, thread_info, thread_index, [&](size_t frame_index, size_t frame_count) {
                     app->event_loop().deferred_invoke([&, frame_index, frame_count] {
+                        if (!window || !window->is_visible())
```

Comment:
This seems like a TOCTOU issue. IIRC there's a way to get a strong reference, which we should probably use here